packages/core/src/accordion/accordion-panel.element.scss

```diff
@@ -6,8 +6,6 @@
 @import './../styles/mixins/mixins';
 
 :host {
-  --border-color: #{$cds-alias-object-border-color};
-  --border-width: #{$cds-alias-object-border-width-100};
```

Comment:
border is something we customize at the accordion level and not the panel. Removing this erroneous code snippet.